### PR TITLE
DM-27414: update for SAL component Rotator -> MTRotator

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: bin/command_rotator.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
```
Comment (wvreeven) [inline bin/command_rotator.py:1]: In the ts_mthexapod project you renamed the run_hexapod.py and command_hexapod.py scripts but you do not rename the command_rotator.py script. I suggest you do to have a homogeneous naming of your scripts. The same applies to the run_rotator.py script, which I see you did not rename either. Don't forget to update the files in the project that refer to these scripts.

File: doc/version_history.rst [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
+* Rename the package from ``ts_rotator`` to ``ts_mtrotator``.
+
+Requires:
+
+* ts_hexrotcomm 0.10
+* ts_salobj 6.1
+* ts_simactuators 1
+* ts_idl 2.2
+* ts_xml 7
+* Rotator IDL files, e.g. made using ``make_idl_files.py Rotator``
```
Comment (wvreeven) [inline doc/version_history.rst:28]: Please fix this to refer to MTRotator twice.

File: tests/test_csc.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ async def test_bin_script(self):
         """Test running from the command line script.
         """
         await self.check_bin_script(
-            name="Rotator",
+            name="MTRotator",
             index=None,
             exe_name="run_rotator.py",
```
Comment (wvreeven) [inline tests/test_csc.py:50]: Don't forget to update this line when you rename the run_rotator.py script to run_mtrotator.py.

**Review by wvreeven — APPROVED** [review]
I only found one important change to do: rename the scripts in the bin directory. Otherwise this looks good to me.

File: bin/command_rotator.py (hunk 1 above)
Comment (r-owen) [reply bin/command_rotator.py:1]: Good catch. I'll make these changes. Thanks!

**Comment by r-owen** [comment]
Thank you for catching those issues. I made the following changes:
* Updated Rotator -> MTRotator for the IDL instructions in the most recent entry in the version history.
* Renamed all bin scripts to change "_rotator" to "_mtrotator".
* Searched for _rotator and changed all instance to _mtrotator. This caught most of the issues you mentioned, including the bin scripts listed in setup.py.

I also looked for _hexapod in ts_hexapod and found and fixed a few that I had missed the first time.

**Pull request merged**